lib/generators/metagem/new_engine/new_engine_generator.rb

```diff
@@ -0,0 +1,102 @@
+require_relative '../gem_config'
+
+class Metagem::NewEngineGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path('templates', __dir__)
+
+  def load_config
+    config.engine = true
+    config.gem_name = file_name
+    config.human_gem_name = ask("Enter engine 'human name':", default: config.human_gem_name)
+    config.summary = ask("Enter the summary of what your plugin does (one sentence):")
+    config.description = ask("Enter engine description (try a little harder here):")
+  end
+
+  def show_info
+    puts config.to_s
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.